app/templates/social_link_form.html

```diff
@@ -0,0 +1,27 @@
+<div class="form-section w-100">
+  {% if col_class %}<div class="col-2 u-text-right" >{% endif %}
+    <label for="socials">Links:</label>
+  {% if col_class %}</div>{% endif %}
+  
+  <div {% if col_class %} class="col-10" style="padding-top:0" {% endif %}>
+    <div class="social-inputs">
+      {% for link in social_links %}
+        <div class="form-group">
+          <select class="form-group-label select btn-grey" name="social-type">
+            {% for type in default_social_links %}
+              <option value="{{ type }}" {% if type == link.link.name %}selected{% endif %}>{{ type }}</option>
+            {% endfor %}
+          </select>
+          <input type="text" class="form-group-input" placeholder="{{ link.link.placeholder }}" value="{{ link.content }}" name="social-content" required>
+          <button class="form-group-btn social-del" disabled>x</button>
```

Comment:
I think a `zmdi-close` icon would look better but I don't know if it's worth the effort right now.